modules/KalturaSupport/kalturaIframeClass.php

```diff
@@ -578,6 +578,17 @@ private function getEntryTitle(){
 		return "Kaltura Embed Player iFrame";
 	}
 
+	function outputCompatibilityTags(){
+		$flashvars = $this->request->getFlashVars();
```

Comment:
please add check to the request (not null) just to be on the safe side @vladm3 
after that we can merge it - Thanks!